AnkiDroid/src/test/java/com/ichi2/anki/DeckPickerCheckDatabaseListenerTest.kt

```diff
@@ -127,43 +126,41 @@ class DeckPickerCheckDatabaseListenerTest : RobolectricTest() {
     }
 
     /**COULD_BE_BETTER: Listener is too coupled to this  */
-    @KotlinCleanup("replace getters with variables")
     private class DeckPickerTestImpl : DeckPicker() {
-        private var mDidDisplayDialogLoadFailed = false
-        private var mDidDisplayMessage = false
-        private var mDidDisplayDbLocked = false
-        fun didDisplayDialogLoadFailed(): Boolean {
-            return mDidDisplayDialogLoadFailed
-        }
+        var didDisplayDialogLoadFailed = false
+            private set(value) {
+                field = value
+            }
```

Comment:
Can this and the below properties be simplified like this?
```suggestion
        private var didDisplayDialogLoadFailed = false
```

Reply:
i already simplified the below properties

Reply:
Just a note, this
```kt
var didDisplayDialogLoadFailed = false
    private set(value) {
        field = value
    }
```
is the same as
```kt
var didDisplayDialogLoadFailed = false
    private set
```

Reply:
ok